openedx/core/djangoapps/user_api/verification_api/serializers.py

```diff
@@ -0,0 +1,20 @@
+""" Verification status API endpoint serializer. """
+
+from rest_framework import serializers
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
+
+
+class SoftwareSecurePhotoVerificationSerializer(serializers.ModelSerializer):
+    expires = serializers.SerializerMethodField()
```

Comment:
Assuming our version of DRF is recent enough:
`expires = serializers.DateTimeField(source='expiration_datetime')`